hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/metadata/SparkHoodieBackedTableMetadataWriter.java

```diff
@@ -161,27 +161,28 @@ protected void commit(String instantTime, Map<MetadataPartitionType, HoodieData<
 
       if (!metadataMetaClient.getActiveTimeline().containsInstant(instantTime)) {
         // if this is a new commit being applied to metadata for the first time
-        writeClient.startCommitWithTime(instantTime);
+        LOG.info("New commit at " + instantTime + " being applied to MDT");
       } else {
```

Comment:
to MDT -> to MDT.

Reply:
Fixed

Reply:
Can we sync the logic also to `FlinkHoodieBackedTableMetadataWriter` ?